<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="2" guice-stage="DEVELOPMENT" name="Failed suite [AllTests (0)]" parallel="methods">
  <listeners>
    <listener class-name="com.epam.mentoring.globoforce.calculator.test.TestListener"/>
  </listeners>
  <test name="Test Double Division by Zero(failed)" parallel="methods">
    <parameter name="a" value="150"/>
    <parameter name="b" value="0"/>
    <classes>
      <class name="com.epam.mentoring.globoforce.calculator.test.DivDoubleTest">
        <methods>
          <include name="createCalculator"/>
          <include name="divDoubleTestException" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.epam.mentoring.globoforce.calculator.test.DivDoubleTest -->
    </classes>
  </test> <!-- Test Double Division by Zero(failed) -->
</suite> <!-- Failed suite [AllTests (0)] -->
